@{
    Layout = null;
}
<!DOCTYPE html>
<html>
<head>
    <meta name="viewport" content="width=device-width" />
    @if (!CloudAppConfig.IsSelfHosted)
    {
        <title>Try - Dashboards | Connection Error</title>
    }
    else
    {
        <title>[[[Database Error]]]</title>
    }

    @await Html.PartialAsync("../Shared/_FontReference")
    <link rel="stylesheet" href="~/bundles/css/layout.css" asp-append-version="true" />
    <link rel="stylesheet" href="~/bundles/css/error-page-new.css" asp-append-version="true" />
    <link rel="stylesheet" href="~/bundles/css/error-page.css" asp-append-version="true" />
    <script src="~/Scripts/jQuery/jquery-1.10.2.min.js"></script>
    <script>
        $(document).ready(function () {
            $(".base-container-bg").css("display", "");
        });
    </script>
</head>
@if (!CloudAppConfig.IsSelfHosted)
{
    <body>
        <div class="base-container-bg" style="display: none">
            <div class="base-conatiner" id="base-conatiner">
                <div id="main-div">
                    <div class="app-logo">
                        <img src="@Url.Content("~/logos?type=login")" onerror="if (this.src != '@Url.Content(CloudAppConfig.FallbackLoginLogoUrl)') this.src = '@Url.Content(CloudAppConfig.FallbackLoginLogoUrl)';">
                    </div>
                    <div class="secondary-div">
                        <div class="page-title">
                            Connection Error
                        </div>
                        <div class="page-content">
                            <span>Error establishing a connection.</span><br />
                            <span class="causes-class">Possible causes:</span><br />
                            <ul class="causes-ul">
                                <li>Server might be down.</li>
                            </ul>
                            <span class="causes-class">Things you may try:</span><br />
                            <ul class="causes-ul">
                                <li>Try again after sometime.</li>
                                <li>If the issue still persist, create a <a href="https://www.syncfusion.com/support/directtrac/incidents/newincident" target="_blank">support incident.</a></li>
                            </ul>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </body>
}
else
{
    <body class="error-full-content">
        <div class="error-page-outer">
            <div class="error-page-content">
                <div class="error-inside">
                    <div class="error-image-center-1">
                        <img src="@CloudAppConfig.SyncfusionLoginLogoUrl" class="error-image-1">
                    </div>
                    <div class="error-image-center-2">
                        <img src="@GlobalAppSettings.CdnDomain/static/error-pages/v1/500.svg" class="error-image-2">
                    </div>
                    <div class="error-type">[[[Database connection error]]]</div>
                    <div class="error-text">
                        [[[Sorry! Server has encountered error while establishing the connection to application database. This error could be occurred due to below possible causes:]]]

                        <ul class="list-style">
                            <li>[[[The database server might have been down.]]]</li>
                            <li>[[[The login credentials for the database might have been changed.]]]</li>
                            <li>[[[The database might have been corrupted.]]]</li>
                        </ul>
                    </div>
                </div>
            </div>
        </div>
        @await Html.PartialAsync("../Shared/_FooterOnPremise")
    </body>
}
</html>